docs/sources/style-guide/write-for-developers/index.md

```diff
@@ -0,0 +1,148 @@
+- If the element does some sort of action, start the first sentence of the description with an action verb. 
+
+## Code examples
+
+It is common for readers of documentation to skim through it to find code samples they can copy and paste and run "as is". For this reason, it is worth taking the time to ensure that whenever possible the examples you provide are production-ready.
+
+Of course, this doesn't mean you should avoid writing code examples intended merely to illustrate a point. When your intent is to explain something, it isn't necessary for your code to be runnable so long as the developer can learn from it to do something similar on their own.
+
+When providing an example, give a written description. You can put it either in the body of the document or as explanatory comments within the example code.
+
+Remember the basic rule: your explanation of the code shouldn't describe _what_ it does, but _why_ it does what it does. (Refer to [_Docs for Developers_](https://docsfordevelopers.com/)).
+
+### Formatting code examples
+
+Here are guidelines to follow when formatting code examples.
+
+- Use spaces, not tabs. 
+- Follow Grafana's accepted [coding style guidelines](https://github.com/grafana/grafana/blob/main/contribute/style-guides/code-comments.md).)
+- Wrap lines at 80 characters.
+- When omitting code, use three dots (...). Don't use the ellipsis character (…).
+
+Introduce each code sample with a sentence or paragraph to establish its context. End the introduction with a colon if it immediately precedes the sample or a period if it doesn't. 
+
+### Paths, filenames and URLs
+
+There are many types of information that should be put in code font. Among these are filenames, paths, folders, and directories. However, don't format domain names or URLs as code if you intend the user to follow the link. 
+
+While there are times when you may want to make a URL clickable, it is better to follow our [guidelines for references]({{< relref "../../../writing-guide/references/" >}}).
+
+## Command lines
+
+Use the following conventions when you include commands on the command line in technical content.
+
+- Do not assume everyone is using Linux. Make sure instructions include enough information for Windows and Mac users to successfully complete procedures.
+
+- Do not add `$` before commands. Make it easy for users to copy and paste commands.
+
+  - **Right:** `sudo yum install grafana`
+  - **Wrong:** `$ sudo yum install grafana`
+
+- Include `sudo` before commands that require `sudo` to work.
+
+For terminal examples and Grafana configuration, use a `bash` code block:
+
+```bash
+sudo yum install grafana
+```
+
+If your command-line instructions include a combination of input and output lines, use separate code blocks for input and output, and use a `console` code block for the output.
```

Comment:
Again, it would be helpful to provide the syntax for how to properly form the console code block in a visible way.

Reply:
See comment above.

Reply:
@jdbaldry, if you have not already been asked to vet this `console` bit, can you see if this is accurate? I believe it is, but want to be sure.